### ISSUE TITLE
Clearcase vobs keyword

### ISSUE THREAD

**Issue by kgiannak** [oracle/opengrok#2752]
I started Opegrok from the latest docker image (Tomcat 9, JRE 8) and configured it to point
a mercurial repository that has many directories as follows:

_/repository/software/vobs/application/1
/repository/software/vobs/application/2
/repository/software/vobs/application/3_

Although the indexing is completed and the source code is searchable the history is not available through the user interface.
It seems that even I use a different SCM, Opengrok's HistoryGuru messes with Clearcase. Discovered that the evaluation of Clearcase includes a [check ](https://github.com/oracle/opengrok/blob/9ce52a219b9e5772102d1fdfcfd7c3c672a332b4/opengrok-indexer/src/main/java/org/opengrok/indexer/history/ClearCaseRepository.java#L246) for the keyword "vobs" in the file paths which appears to be related.

**Comment by idodeclare** [comment]
One workaround would be to lower the scanning depth from its default of 3 by adding `--depth 1` to your indexing arguments.

**Comment by idodeclare** [comment]
Probably we need to make the ClearCase check actually do an `cleartool lscheckout` after finding `vobs` so it doesn’t inadvertently activate.

**Comment by vladak** [comment]
Yep, this is a bug. Arbitrary directory named `vobs` should not result in ClearCase repository detection.

**Comment by vladak** [comment]
It would be nice if someone with ClearCase experience could fix this.

**Comment by idodeclare** [comment]
@kgiannak , another workaround will be if PR #2969 is merged. Then you could `--disableRepository clearcase` so that "vobs" is not recognized as a marker for ClearCase repositories.
